utils/sl/check_script.py

```diff
@@ -110,19 +105,16 @@ def write_variables(outfile, prefix, variables):
         for variable in variables:
             if variable.upper() == 'PREREQS':
                 newlines += 4
-                outfile.write("%sfrom prereq_tools import PreReqComponent\n"
-                              % prefix)
-                outfile.write("%sscons_temp_env = DefaultEnvironment()\n"
-                              % prefix)
+                outfile.write("%sfrom prereq_tools import PreReqComponent\n" % prefix)
+                outfile.write("%sscons_temp_env = DefaultEnvironment()\n" % prefix)
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string